week3_model_free/qlearning.ipynb

```diff
@@ -209,18 +209,18 @@
     "    This function should \n",
     "    - run a full game, actions given by agent's e-greedy policy\n",
     "    - train agent using agent.update(...) whenever it is possible\n",
-    "    - return total reward\n",
+    "    - return the total reward\n",
     "    \"\"\"\n",
     "    total_reward = 0.0\n",
     "    s = env.reset()\n",
     "\n",
     "    for t in range(t_max):\n",
-    "        # get agent to pick action given state s.\n",
+    "        # get an agent to pick action given state s.\n",
```

Comment:
I think here we should use "the". Furthermore, in my opinion, a better way to phrase this would be:

> Get an action from the agent given state s